managed-services/transactional-email/quickstart.mdx

```diff
@@ -26,9 +26,9 @@ In this quickstart, we show you how to configure your domain with Transactional
 ## How to configure your domain with Transactional Email
 
 1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. The **Transactional Email** overview page displays.
-2. Click **Add a Domain**.
+2. Click **Add Domain**.
 3. Enter the domain name you want to use with Transactional Email.
-4. Click **Validate domain name** to confirm.
+4. Tick the box to acknowledge Scaleway’s antispam policy, then, click **Validate domain name** to confirm.
```

Comment:
I feel like saying "tick the box to acknowledge Scaleway's anti spam policy" is not really correct. We ask them to read it and accept it because if they end up using TEM to send marketing emails or spam, they cannot say they didn't know it was not meant for that purpose. Hence my previous suggestion :)

Reply:
woops, misread !